### ISSUE TITLE
Resource opening tracking and sync with community

### ISSUE THREAD

**Issue by dogi** [open-learning-exchange/take-home#24]

**Comment by timecomplexity** [comment]
Is this a duplicate of the combination of https://github.com/open-learning-exchange/take-home/issues/23 and https://github.com/open-learning-exchange/take-home/issues/46 ?
If so, we could probably go ahead and close this just so there's no confusion about that.

**Comment by leomaxi** [comment]
No. https://github.com/open-learning-exchange/take-home/issues/24 -: This is tracking usage data plus rating, user logs generated when making use of resources/materials in "my library" whiles https://github.com/open-learning-exchange/take-home/issues/46 -: Is trying to have the list of resources / materials on a community "Library" listed on the offline (take-home) app
